lib/service/cooperative/ChainSwapSigner.ts

```diff
@@ -95,6 +100,15 @@ class ChainSwapSigner extends CoopSignerBase<{ claim: ChainSwapInfo }> {
         throw Errors.CURRENCY_NOT_UTXO_BASED();
       }
 
+      if (this.disableCooperative) {
+        this.logger.debug(
+          `Cooperative signatures are disabled, not creating partial signature for refund of ${swapTypeToPrettyString(swap.type)} Swap ${swap.id}`,
+        );
+        throw Errors.NOT_ELIGIBLE_FOR_COOPERATIVE_REFUND(
+          'cooperative refunds are disabled',
```

Comment:
```suggestion
          'cooperative signatures are disabled',
```

And let's put that into a constant